tests/basic/pool_function_test.go

```diff
@@ -84,6 +85,33 @@ var _ = Describe("{PoolExpandSmoky}", func() {
 		dash.VerifyFatal(resizeErr, nil, "Pool expansion does not result in error")
 		verifyPoolSizeEqualOrLargerThanExpected(poolIDToResize, targetSizeGiB)
 	})
+
+	// test resizing all pools at the same time and ensure only the first one makes progress
+	It("Select all pools and expand them concurrently. ", func() {
+		StartTorpedoTest("PoolExpandRejectConcurrent",
+			"Validate storage pool expansion rejects concurrent requests", nil, 0)
+		// collect all pools available
+		var pools []*api.StoragePool
+		nodes := node.GetNodes()
```

Comment:
Please use node.GetStorageDriverNodes, node.GetNodes will get all the nodes including master node and nodes where portworx is not enabled. This will cause failures while running in cloud environments where we deploy a node pool for running torpedo